internal/pkg/manifest/validate_test.go

```diff
@@ -178,12 +178,41 @@ func TestLoadBalancedWebService_validate(t *testing.T) {
 						},
 					},
 					NLBConfig: NetworkLoadBalancerConfiguration{
-						Port:            aws.String("443"),
-						TargetContainer: aws.String("foo"),
+						MainListener: NetworkLoadBalancerListener{
+							Port:            aws.String("443"),
+							TargetContainer: aws.String("foo"),
+						},
+					},
+				},
+			},
+			wantedErrorMsgPrefix: `validate target for nlb.listener: `,
+		},
+		"error if fail to validate network load balancer target for additional listener": {
```

Comment:
Can you remind me why these configurations are problemetic 😰 Are any containers exposing the same ports?

Reply:
Ohh no, if you look at `additional_listeners` it is trying to expose port `444` of sidecar container `foo` which does not even exist that is why we are getting error - `validate target for nlb.additional_listeners[0]: target container "foo" doesn't exist`